core/src/settings.ts

```diff
@@ -99,6 +99,15 @@ export const L2_WORKER_VRF_FULFILL_QUEUE_NAME = `${DEPLOYMENT_NAME}-worker-vrf-f
 export const L2_WORKER_REQUEST_RESPONSE_REQUEST_QUEUE_NAME = `${DEPLOYMENT_NAME}-worker-request-response-request-l2-queue`
 export const L2_WORKER_REQUEST_RESPONSE_FULFILL_QUEUE_NAME = `${DEPLOYMENT_NAME}-worker-request-response-fulfill-l2-queue`
 
+export const NONCE_MANAGER_REQUEST_RESPONSE_QUEUE_NAME = `${DEPLOYMENT_NAME}-nonce-manager-request-response-queue`
+export const NONCE_MANAGER_VRF_QUEUE_NAME = `${DEPLOYMENT_NAME}-nonce-manager-vrf-queue`
+export const NONCE_MANAGER_L2_DATA_FEED_QUEUE_NAME = `${DEPLOYMENT_NAME}-nonce-manager-data-feed-l2-queue`
+export const NONCE_MANAGER_DATA_FEED_QUEUE_NAME = `${DEPLOYMENT_NAME}-nonce-manager-data-feed-queue`
+export const NONCE_MANAGER_L2_REQUEST_RESPONSE_FUFILL_QUEUE_NAME = `${DEPLOYMENT_NAME}-nonce-manager-request-response-l2-fulfill-queue`
+export const NONCE_MANAGER_L2_REQUEST_RESPONSE_REQUEST_QUEUE_NAME = `${DEPLOYMENT_NAME}-nonce-manager-request-response-l2-request-queue`
+export const NONCE_MANAGER_L2_VRF_FULFILL_QUEUE_NAME = `${DEPLOYMENT_NAME}-nonce-manager-vrf-l2-fulfill-queue`
+export const NONCE_MANAGER_L2_VRF_REQUEST_QUEUE_NAME = `${DEPLOYMENT_NAME}-nonce-manager-vrf-l2-request-queue`
+
```

Comment:
would let's remove these updates

Reply:
left an explanation above as well, but very briefly, i want to maintain these files as well. If we exclude them from maintenance then it's better to delete them. Because it'll be challenging to bring them up to speed if we decide to use them in the future, especially in the far future.

Reply:
let's remove all datafeed queue names here 🙏